resources/docker-images/build.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+. "${THIS_SCRIPT%/*}/../../resources/build/builder.inc.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+################################ Main script ################################
+
+. "${KEYMAN_ROOT}/resources/build/minimum-versions.inc.sh"
+
+builder_describe \
+  "Build docker images" \
+  ":android" \
+  ":base" \
+  ":core" \
+  ":linux" \
+  ":web" \
+  "--ubuntu-version=UBUNTU_VERSION  The Ubuntu version (default: ${KEYMAN_DEFAULT_VERSION_UBUNTU_CONTAINER})" \
+  "--node=NODE_MAJOR                Node version (default: ${KEYMAN_MIN_VERSION_NODE_MAJOR})" \
+  "--emscripten=EMSCRIPTEN_VERSION  Emscripten version (default: ${KEYMAN_MIN_VERSION_EMSCRIPTEN})" \
```

Comment:
I think we should use the `KEYMAN_USE_NVM` and `KEYMAN_USE_EMSDK` variables, and not make these user-configurable.

Reply:
Yeah, we can create a script that calls `shellHelperFunctions.sh:verify_npm_setup()` and `locate_emscripten.inc.sh:verify_emscripten_version()` to do the prep work. Then the future `nvm` and emsdk calls are very quick because they are really no-ops.